blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+date: 2025-07-25
+author: Kaarthis
+categories: general
+---
+
+In London at KubeCon EU 2025, AKS announced our expansion of what AKS Long Term Support (LTS) includes. Today, we're sharing more details about this offering that addresses one of the most critical challenges enterprises face when running Kubernetes at scale.
+From our conversations with customers, we consistently hear the same concerns: *"How do I balance Kubernetes innovation with the stability my business-critical applications require?"* and *"Why do I need to upgrade my clusters so frequently when my applications are running perfectly?"*
+
+On the flip side, customers also ask: *"If I don't upgrade frequently, how do I still ensure that I'm getting security fixes and ecosystem compatibility updates for my Kubernetes infrastructure?"* AKS LTS directly addresses these real-world challenges.
+
+## Why AKS LTS Matters
+
+Enterprise organizations running mission-critical workloads on Kubernetes have consistently told us they need more predictable, stable platform foundations. While the rapid innovation pace of Kubernetes is fantastic for driving new capabilities, it can create challenges for organizations that require:
+
+- **Extended support lifecycles** that align with enterprise planning cycles
+- **Predictable upgrade windows** that minimize operational disruption
+- **Stability guarantees** for production workloads that can't tolerate frequent changes
+- **Compliance requirements** that demand long-term support commitments
+
+AKS LTS directly addresses these needs by providing a **support plan** specifically designed for enterprise stability requirements.
+
+## What is AKS LTS?
+
+AKS Long Term Support is a **support plan** that provides **24 months** of support for Kubernetes versions from their GA date in AKS, compared to the standard 12-15 month support lifecycle. **Here's the key change: Every currently supported AKS Kubernetes version is now also available for long term support (LTS).**
+
+This isn't just extended maintenance—it's a comprehensive support commitment designed for enterprises running mission-critical workloads.
+
+**The Real Impact:** Instead of planning cluster upgrades every 12-15 months, you can now plan major Kubernetes upgrades every 24 months. This **reduces upgrade frequency by 50%** and can **save enterprises 40+ hours of operational overhead annually** per cluster while maintaining full security and support coverage.
+
+Key characteristics of AKS LTS include:
+
+- **Extended Support Timeline:** 24 months of full support from GA date, including security patches and critical bug fixes
+- **Predictable Release Cadence:** New LTS versions released annually, allowing for planned upgrade cycles
+- **Enterprise-Grade Stability:** Focus on proven, stable features rather than frequent changes
+- **Backwards Compatibility:** Strong commitment to API stability and workload compatibility within LTS versions
```

Comment:
these two would be true for community supported versions too no? just that the window of time is different

Reply:
yes i can add as is the case with community versions but with extended support timeline good point

Reply:
also cost is another.